### ISSUE TITLE
providing a xlsx-template

### ISSUE THREAD

**Issue by fdiehr** [funkyfuture/m4p0-rs-metadata-import#17]
Für die Partner habe ja ich ein XSLX-Template erstellt, dass sie für die Angabe der Metadaten nutzen sollen. Es muss im Excel-Format sein, da bei CSV z.B. Dropdownfunktion für vordefinierte Werte nicht funktionieren. Dazu kommt, dass die Partner eher gewohnt sind mit Excel zu arbeiten.
Das erfordert das anschließende konvertieren in CSV. Ist das Teil der Importroutine? Bisher nicht, oder? Würde derjenige, der dann den Import macht, diese Konversion vornehmen? Wenn ja, wäre es dann sinnvoll noch etwas zur Konversion in die Doku aufzunehmen?

Hier das Template, mit dem wir für den Test gearbeitet haben. 
[TEMPLATE-ImportDigitaleObjekte.xlsx](https://github.com/funkyfuture/m4p0-rs-metadata-import/files/4102400/TEMPLATE-ImportDigitaleObjekte.xlsx)

**Comment by funkyfuture** [comment]
> Wenn ja, wäre es dann sinnvoll noch etwas zur Konversion in die Doku aufzunehmen?

das ist der plan.

**Comment by fdiehr** [comment]
ah topp, okay. dann kann ich das template genau so bereitstellen. super!

**Comment by funkyfuture** [comment]
genau, wenn du mir noch ne url mitteilst, kann ich die auch in die doku aufnehemn.
